library/libnames.ml

```diff
@@ -156,10 +156,15 @@ let qualid_of_dirpath dir =
 
 type object_name = full_path * KerName.t
 
-type object_prefix = DirPath.t * (ModPath.t * DirPath.t)
+type object_prefix = {
+  obj_dir : DirPath.t;
+  obj_mp  : ModPath.t;
+  obj_sec : DirPath.t;
```

Comment:
Could we add a comment saying what this is? In particular, about how the two `DirPath.t` are related.

Reply:
I meant in the `.mli` probably.

Reply:
@herbelin @ppedrot do you know that those fields are for?